packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -0,0 +1,601 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import {describe, it, beforeEach} from 'mocha';
+import assert from 'assert';
+import {MockSubscription} from './mocks/mock-pubsub-subscription';
+import {PubSub} from '@google-cloud/pubsub';
+import {MockFirestore, MockRecord} from './mocks/mock-firestore';
+import {CloudLogsProcessor} from '../../../src/data-processors/cloud-logs-data-processor';
+import {loadFixture} from './util/test-util';
+import {ObjectWritableMock} from 'stream-mock';
+import pino from 'pino';
+
+let mockSubscription: MockSubscription;
+let mockFirestore: MockFirestore;
+let processor: CloudLogsProcessor;
+
+const VALID_MESSAGES: {[name: string]: {}} = loadFixture(
```

Comment:
I might split each of these into their own file & constant - the aggregate of these don't seem to mean anything. Also, it might make sense to load it closer to the test it's used in to make the individual tests easier to read.

Reply:
Sure, I can do that